code/modules/clothing/under/miscellaneous.dm

```diff
@@ -1,3 +1,13 @@
+
+/obj/item/clothing/under/banearmour
+    name = "banearmour"
+    desc = "You merely adopted the darknes,I was born in it.."
```

Comment:
Also, an ellipsis should have three dots and another one if they're the end of a sentence.